zanata-war/src/main/java/org/zanata/webtrans/client/util/DateUtil.java

```diff
@@ -53,8 +44,7 @@ public static String formatTime(Date date) {
      */
     public static String formatLongDateTime(Date date) {
         if (date != null) {
-            return DateTimeFormat.getFormat(DATE_TIME_LONG_PATTERN)
-                    .format(date);
+            return dtfLong.format(date);
```

Comment:
Why did we change it to "not thread safe"?
From what I can see, it was creating its own formatter before which made it thread safe.